stroeer/web/article/v1/web_article.proto

```diff
@@ -4,6 +4,7 @@ package stroeer.web.article.v1;
 
 import "stroeer/core/v1/article.proto";
 
+option go_package = "github.com/stroeer/tapir/web/article/v1;article";
```

Comment:
@moritzzimmer Shouldn't this resolve the generate go files from `protoc`'s `--go_out` directory?

```suggestion
option go_package = "github.com/stroeer/tapir/go/stroeer/web/article/v1;article";
```